test/integration/apiserver/patch_test.go

```diff
@@ -72,7 +70,7 @@ func TestPatchConflicts(t *testing.T) {
 				Namespace(ns.Name).
 				Resource("secrets").
 				Name("test").
-				Body([]byte(fmt.Sprintf(`{"metadata":{"labels":{"%s":"%s"}, "annotations":{"$patch":"replace","%s":"%s"}}}`, labelName, value, annotationName, value))).
+				Body([]byte(fmt.Sprintf(`{"metadata":{"labels":{"%s":"%s"}, "annotations":{"%s":"%s"}}}`, labelName, value, annotationName, value))).
```

Comment:
> to prevent regressions where the directives are not honored in retries

To exercise this, we need to figure out a way to make the `"$patch":"replace","` patch races with other non-conflicting patches.